scripts/configurator/parser/cmake_parser.py

```diff
@@ -5,7 +5,7 @@ def read_file(filepath):
    with open(filepath, 'r') as f:
       return f.readlines()
 
-def lines_to_rows(file_lines: List[str]) -> List[cmake_row]:
+def parse_variables(file_lines: List[str]) -> List[cmake_row]:
```

Comment:
`parse_lines` would be better. We're creating row objects as a results, not just variables.

Reply:
Or `parse_rows` seems good as well. Try to thing very carefully what's the input and what's the output. The name should be as descriptive as possible.